### ISSUE TITLE
Category and CategoryBase should be in separate apps

### ISSUE THREAD

**Issue by winniehell** [callowayproject/django-categories#55]
**long version**
After a few hours of bug hunting, I found out why my current translation is not loaded. I think, there is a problem with your directory structure:

For each app django loads the translations from the locale subdirectory. In my case, I only load the categories.editor app - so I only get the translations in categories/editor/locale. The problem is that categories.editor relies on stuff from the categories folder and those translations won't get loaded.

So did I get something wrong or should CategoryBase rather be a model in categories.editor (or something like categories.common) in order to be translated?

**short version**
- to have the CategoryBase model translated its app has to be in INSTALLED_APPS
- if one does not want the Category model, its app should not be in the INSTALLED_APPS

=> CategoryBase and Category should not be in the same app

**current workaround**
If you only want categories.editor put categories and categories.editor in INSTALLED_APPS and then unregister the Admin for Category:

    admin.site.unregister(Category)
